### PR TITLE
Fixed cell alignment issue #674 and remove key configuration

### PR DESCRIPTION
**Pull request by nxanil**
@bmukheja @ronakmshah 

Fixed issue for cell width if any empty cell present.
Also,  `key` configuration no more required. It is irrelevant now.

Please have a look.

### CODE REVIEW

**Comment by ronakmshah** [comment]
issue #674

**Comment by nxanil** [comment]
@natabal Related to issue #674 Please review the changes.

File: src/components/Visualization/index.js [hunk 1]
```diff
@@ -216,13 +216,13 @@ class VisualizationView extends React.Component {
                             let graphQueryParams = {};
                             let resetFilters = false;
 
-                            if(configuration.key) {
+                            if(configuration.graph === 'HeatmapGraph') {
```
Comment (natabal) [inline src/components/Visualization/index.js]: This should be handled in a configuration for the HeatmapGraph! Similarly to how we have done for table

File: src/components/Visualization/index.js [hunk 2]
```diff
@@ -198,16 +197,18 @@ class VisualizationView extends React.Component {
...
-                                if(this.props.orgContext[vizID] === vKey)
-                                    resetFilters = true;
+                            let graphQueryParams = {},
+                                resetFilters = false,
+                                vizID = `${id.replace(/-/g, '')}vkey`,
+                                vKey = GraphComponent.getGraphKey(configuration);
 
-                                graphQueryParams[vizID] = vKey;
+                            if(vKey) {
+                                if(this.props.orgContext[vizID] === vKey(d)) {
```
Comment (natabal) [inline src/components/Visualization/index.js]: instead of calling vKey(d) twice let's cache the result and use in both case: if statement and below.

**Pull request merged**